### ISSUE TITLE
The process finishes upon starting without any error or GUI shown

### ISSUE THREAD

**Issue by Subdartha** [denizsafak/AutoSubSync#4]
Hi,
I've tried running your script versions 2.5 and 2.6 but the result is the same: no GUI appears, the process just finishes without any errors.
Goes like this:
`D:\Distr\Video\AutoSubSync-v2.5>run.bat
Creating virtual environment...
Activating virtual environment...
Checking the requirements...
Installing package: chardet==5.2.0
WARNING: You are using pip version 21.2.3; however, version 24.3.1 is available.
You should consider upgrading via the 'D:\Distr\Video\AutoSubSync-v2.5\.venv\Scripts\python.exe -m pip install --upgrade pip' command.
Installing package: tkinterdnd2==0.4.2
WARNING: You are using pip version 21.2.3; however, version 24.3.1 is available.
You should consider upgrading via the 'D:\Distr\Video\AutoSubSync-v2.5\.venv\Scripts\python.exe -m pip install --upgrade pip' command.
Installing package: ffsubsync==0.4.26
WARNING: You are using pip version 21.2.3; however, version 24.3.1 is available.
You should consider upgrading via the 'D:\Distr\Video\AutoSubSync-v2.5\.venv\Scripts\python.exe -m pip install --upgrade pip' command.
Installing package: setuptools==75.3.0
WARNING: You are using pip version 21.2.3; however, version 24.3.1 is available.
You should consider upgrading via the 'D:\Distr\Video\AutoSubSync-v2.5\.venv\Scripts\python.exe -m pip install --upgrade pip' command.
Starting AutoSubSync...

D:\Distr\Video\AutoSubSync-v2.5>`
The results on v2.6 and v2.5 are identical.

I run Windows 10 Pro 21H2 build 19044.2251.

**Comment by denizsafak** [comment]
The Python installed on your computer may be an old version. May I know the Python version you are currently using? I will test it myself. And can you test it with Python 3.12.7?

**Comment by Subdartha** [comment]
I'm using python version 3.10.0. Let me upgrade my python to 3.12.7 and see what happens.

**Comment by Subdartha** [comment]
It's the same with the new version of Python.
`D:\Distr\Video\AutoSubSync-v2.6>run.bat
Creating virtual environment...
Activating virtual environment...
Checking the requirements...
Installing package: chardet==5.2.0

[notice] A new release of pip is available: 24.2 -> 24.3.1
[notice] To update, run: python.exe -m pip install --upgrade pip
Installing package: tkinterdnd2==0.4.2

[notice] A new release of pip is available: 24.2 -> 24.3.1
[notice] To update, run: python.exe -m pip install --upgrade pip
Installing package: ffsubsync==0.4.26

[notice] A new release of pip is available: 24.2 -> 24.3.1
[notice] To update, run: python.exe -m pip install --upgrade pip
Installing package: setuptools==75.3.0

[notice] A new release of pip is available: 24.2 -> 24.3.1
[notice] To update, run: python.exe -m pip install --upgrade pip
Starting AutoSubSync...

D:\Distr\Video\AutoSubSync-v2.6>`

**Comment by denizsafak** [comment]
Interesting. I need to see some output to determine the problem. Can you try this in CMD:

```CMD
cd D:\Distr\Video\AutoSubSync-v2.6
.venv\Scripts\python.exe main\AutoSubSync.pyw
```
and give me the output?

**Comment by Subdartha** [comment]
I got this:
`D:\Distr\Video\AutoSubSync-v2.6>pip install tkinter
ERROR: Could not find a version that satisfies the requirement tkinter (from versions: none)
ERROR: No matching distribution found for tkinter`
So I obviously installed Python without "Install TCL/Tk and IDLE ide" option. After I modified the installation, I'm getting the GUI.

There is another error now when I try to do the sync, but I'll open a new issue for this in a minute.
Thanks a lot! :)
